extras/minimal-printf/Makefile

```diff
@@ -1,5 +1,5 @@
 # Just for manual testing
 .PHONY: test
 test: duk_minimal_printf.c
-	gcc -fno-stack-protector -m32 -otest -Wall -Wextra -Os -fomit-frame-pointer duk_minimal_printf.c test.c
+	gcc -std=c99 -Wall -Wextra -fno-stack-protector -m32 -Os -fomit-frame-pointer -otest duk_minimal_printf.c test.c
```

Comment:
With the minimal printf I was trying to see what options matter re: the footprint it compiles down to.

These Makefiles are pretty placeholdery; I added the warnings options to catch obvious issues, but they're nowhere near good. For proper coverage there would also need to be a clang compilation, C++ compilation, etc.